base/special/hyperbolic.jl

```diff
@@ -50,7 +50,7 @@ function sinh_kernel(x::Float64)
     x2 = x*x
     x2lo = fma(x,x,-x2)
     hi_order = evalpoly(x2, (8.333333333336817e-3, 1.9841269840165435e-4,
-                             2.7557319381151335e-6, 2.5052096530035283e-8,
+			     2.7557319381151335e-6, 2.5052096530035283e-8,
```

Comment:
Looks like this is just a whitespace change; can it be removed?

Reply:
Yeah. Total accident.